pylds/base_discrete.py

```diff
@@ -26,6 +27,109 @@ def check_if_points_escape_box(u, box_boundaries):
     u_indices = (x >= box_x_min) & (x <= box_x_max) & (y >= box_y_min) & (y <= box_y_max)
     return u_indices
 
+def pbc_correction_coords_single_axis(x, box_origin, box_length):
+    """
+    Correct coordinate in a single axis by Periodic Boundary Conditions (PBCs).
+
+    Parameters
+    ----------
+    x : array_like, shape(n, )
+        points in axis to correct by PBCs
+    
+    origin : array
+        
+    box_length : array
+        
+    Returns
+    -------
+    x_pbc : array_like, shape(n, )
+        array of corrected points for periodic box
+    """
+    x_pbc = x
+    x0 = box_origin
+    L = box_length
```

Comment:
Matter of preference, but box_origin and box_length are not too long to be used as variable names

Reply:
That's right. Not too long for readability, but long enough for coding formulas where these variables are used many times. 
I found it more practical, and less prone to typos.